### PR TITLE
[bitnami/cassandra] Allow adding custom configuration files

### PR DESCRIPTION
**Pull request by javsalgar**
Signed-off-by: Javier J. Salmeron Garcia <jsalmeron@bitnami.com>

**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

**Additional information**

### CODE REVIEW

File: bitnami/cassandra/templates/configuration-cm.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{{ if and (.Files.Glob "files/conf/*") (not .Values.existingConfiguration) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "cassandra.fullname" . }}-configuration
```
Comment (juan131) [inline bitnami/cassandra/templates/configuration-cm.yaml]: Shouldn't you use `{{ template "cassandra.configurationCM" . }}`?

File: bitnami/cassandra/templates/configuration-cm.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{{ if and (.Files.Glob "files/conf/*") (not .Values.existingConfiguration) }}
```
Comment (juan131) [inline bitnami/cassandra/templates/configuration-cm.yaml]: I don't understand very well this.. Shouldn't it be?
```
{{ if and (.Files.Glob "files/conf/*") .Values.existingConfiguration }}
```

File: bitnami/cassandra/templates/statefulset.yaml [hunk 3]
```diff
@@ -183,8 +187,13 @@ spec:
           initialDelaySeconds: 20
           timeoutSeconds: 45
 {{- end }}
-{{- if not .Values.persistence.enabled }}
       volumes:
+    {{- if or (.Files.Glob "files/conf/*") .Values.existingConfiguration }}
```
Comment (juan131) [inline bitnami/cassandra/templates/statefulset.yaml]: Is it well indented?

File: bitnami/cassandra/templates/configuration-cm.yaml (hunk 1 above)
Comment (javsalgar) [reply bitnami/cassandra/templates/configuration-cm.yaml]: No, that's for obtaining whether to use the one we generate or the one provided in values.yaml

File: bitnami/cassandra/templates/statefulset.yaml (hunk 3 above)
Comment (javsalgar) [reply bitnami/cassandra/templates/statefulset.yaml]: Fixed

**Pull request merged**